es/refresher-linear-algebra.md

```diff
@@ -1,339 +1,341 @@
 
 <br>
 
 **45. Spectral theorem ― Let A∈Rn×n. If A is symmetric, then A is diagonalizable by a real orthogonal matrix U∈Rn×n. By noting Λ=diag(λ1,...,λn), we have:**
 
-&#10230;
+&#10230; Teoréma espectrasl ― Sea A∈Rn×n. si A es simétrica, entonces A es diagonalizable por una matriz real ortogonal U∈Rn×n.
+Notando Λ=diag(λ1,...,λn), se tiene que:
 
 <br>
 
 **46. diagonal**
 
-&#10230;
+&#10230; diagonal
 
 <br>
 
 **47. Singular-value decomposition ― For a given matrix A of dimensions m×n, the singular-value decomposition (SVD) is a factorization technique that guarantees the existence of U m×m unitary, Σ m×n diagonal and V n×n unitary matrices, such that:**
 
-&#10230;
+&#10230; Descomposición de valores singulares ― Para una mtraiz A de dimensiones m×n, la descomposición en valores singulares (SVD) es una técnica de factorizacion que garantiza que existen matrices U m×m unitaria, Σ m×n diagonal y V n×n unitaria, tal que:
 
 <br>
 
 **48. Matrix calculus**
-
-&#10230;
+ 
+&#10230; Calculo de matrices
 
 <br>
 
 **49. Gradient ― Let f:Rm×n→R be a function and A∈Rm×n be a matrix. The gradient of f with respect to A is a m×n matrix, noted ∇Af(A), such that:**
 
-&#10230;
+&#10230; Gradiente ― Sea f:Rm×n→R una función y A∈Rm×n una matriz. El gradiente de f con respecto a A es una matriz de m×n, notando que ∇Af(A), tal que:
 
 <br>
 
 **50. Remark: the gradient of f is only defined when f is a function that returns a scalar.**
 
-&#10230;
+&#10230; Nota: el gradiente de f esta solo definido cuando f es una función cuyo resultado es un escalar.
 
 <br>
 
 **51. Hessian ― Let f:Rn→R be a function and x∈Rn be a vector. The hessian of f with respect to x is a n×n symmetric matrix, noted ∇2xf(x), such that:**
 
-&#10230;
+&#10230; Matriz Hessiana ― Sea f:Rn→R una función y x∈Rn un vector. La matriz hessiana o hessiano de f con recpecto a x
+es una matriz simétrica de n×n, para ∇2xf(x), tal que:
 
 <br>
 
 **52. Remark: the hessian of f is only defined when f is a function that returns a scalar**
 
-&#10230;
+&#10230; Nota: la matriz hessiana de f solo esta definida cuando f es una función que regresa un escalar
 
 <br>
 
 **53. Gradient operations ― For matrices A,B,C, the following gradient properties are worth having in mind:**
 
-&#10230;
+&#10230; Operaciones de gradiente ― Para matrices A,B,C, las siguientes propiedades del gradiente vale la pena tener en cuenta:
 
 <br>
 
 **54. [General notations, Definitions, Main matrices]**
 
-&#10230;
+&#10230; [Notaciones generales, Definiciones, Matrices principales]
 
 <br>
 
 **55. [Matrix operations, Multiplication, Other operations]**
 
-&#10230;
+&#10230; [Operaciones matriciales, Multiplicación, Otras operaciones]
 
 <br>
 
 **56. [Matrix properties, Norm, Eigenvalue/Eigenvector, Singular-value decomposition]**
 
-&#10230;
+&#10230; [Propiedades matriciales, Norma, Eigenvalor/Eigenvector, Descomposición de valores singulares]
```

Comment:
"Eigenvalue" se traduce como autovalor. "Eigenvector" se traduce como autovector.

Reply:
@fdiazgon at least in some parts of Central América and México, Eigenvalor and Eigenvector is used. Can we use Autovalor (Eigen-valor) or something similar? As many people might be familiar tu this term.

Reply:
I would try to avoid anglicisms when we have a word that exactly matches the meaning: [Vector propio y valor propio](https://es.wikipedia.org/wiki/Vector_propio_y_valor_propio). But if you think it is better to keep it, it's OK.

Reply:
@shervinea  Any opinion regarding this? Any other Spanish speakers want to state their opinion? @gustavovelascoh